README.md

```diff
@@ -53,77 +54,83 @@ scalar.db.storage=grpc
 # The type of the transaction manager
 scalar.db.transaction_manager=grpc
 ```
-- Setting up Scalar DB schema: As mentioned in [`Getting started with Scalar DB`](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb.md) storing and retrieving data in `Scalar DB` can be done using either the `storage` or the `transaction` mode.
-    - Our storage schema is  [`emoney-storage-schema.json`](./src/main/resources/emoney-storage-schema.json)
-    ```
-    {
-      "emoney_storage.account": {
-        "transaction": false,
-        "partition-key": [
-          "id"
-        ],
-        "clustering-key": [],
-        "columns": {
-          "id": "TEXT",
-          "balance": "INT"
-        }
-      }
+Setting up Scalar DB schema: As mentioned in [Getting started with Scalar DB](https://github.com/scalar-labs/scalardb/blob/master/docs/getting-started-with-scalardb.md) storing and retrieving data in `Scalar DB` can be done using either the `storage` or the `transaction` mode.
+
+Now you apply the database schema of the sample application as shown below. (It is stored in [emoney-storage-schema.json](./src/main/resources/emoney-storage-schema.json)).
+Note that this case uses non-transactional mode (storage mode) since `transaction` key is set to false.
+      
+```
+{
+  "emoney_storage.account": {
+    "transaction": false,
+    "partition-key": [
+      "id"
+    ],
+    "clustering-key": [],
+    "columns": {
+      "id": "TEXT",
+      "balance": "INT"
     }
-    ```
-    to register schema to our Cassandra instance we run the following command:
-    ```
-    java -jar tools/scalar-schema-standalone-3.1.0.jar --cassandra -h localhost -u cassandra -p cassandra -f src/main/resources/emoney-storage-schema.json
-    ```
-    Please note that `scalar-schema-standalone-3.1.0.jar` is the schema registration tool that can be found in [`release`](https://github.com/scalar-labs/scalardb/releases) of `Scalar DB`
-
-    - In case using transaction, our schema looks like [`emoney-transaction-schema.json`](src/main/resources/emoney-transaction-schema.json)
-    ```
-    {
-      "emoney_transaction.account": {
-        "transaction": true,
-        "partition-key": [
-          "id"
-        ],
-        "clustering-key": [],
-        "columns": {
-          "id": "TEXT",
-          "balance": "INT"
-        }
-      }
+  }
+}
+```
+You then apply the schema with the following command. Please note that `scalar-schema-standalone-<version>.jar` is the schema registration tool that can be found in [release](https://github.com/scalar-labs/scalardb/releases) of Scalar DB. Please select 3.1.0 or higher since Scalar DB server support is added in 3.1.0.
```

Comment:
```suggestion
You then apply the schema with the following command. Please note that `scalar-schema-standalone-<version>.jar` is the schema tool that can be found in [releases](https://github.com/scalar-labs/scalardb/releases) of Scalar DB. Please select 3.1.0 or higher since Scalar DB server support is added in 3.1.0.
```